core/modules/compare/store/plugin.ts

```diff
@@ -5,7 +5,7 @@ import { Logger } from '@vue-storefront/core/lib/logger'
 export function plugin (mutation, state) {
   const type = mutation.type
 
-  if (type.startsWith(types.SN_COMPARE)) { // check if this mutation is comapre related
+  if (type === types.SN_COMPARE  + '/' + types.COMPARE_ADD_ITEM || types.SN_COMPARE + '/' + type === types.COMPARE_DEL_ITEM) { // check if this mutation is comapre related
```

Comment:
could we use more readable comparison?
something like `type.includes(types.COMPARE_ADD_ITEM) || type.includes(types.COMPARE_DEL_ITEM)`
or `[types.COMPARE_ADD_ITEM, types.COMPARE_DEL_ITEM].some(typeName => type.includes(typeName))`